core/src/main/java/io/questdb/std/Numbers.java

```diff
@@ -899,6 +900,94 @@ public static Long256Impl parseLong256(CharSequence text, int len, Long256Impl l
         return extractLong256(text, len, long256) ? long256 : Long256Impl.NULL_LONG256;
     }
 
+    public static long parseLongDuration(CharSequence sequence) throws NumericException {
+        int lim = sequence.length();
+        if (lim == 0) {
+            throw NumericException.INSTANCE;
+        }
+
+        boolean negative = sequence.charAt(0) == '-';
+        if(negative){
```

Comment:
No proper formatting here as well:
```suggestion
        if (negative){
```